homeassistant/components/splunk/__init__.py

```diff
@@ -75,13 +58,11 @@ def setup(hass, config):
     name = conf.get(CONF_NAME)
     entity_filter = conf[CONF_FILTER]
 
-    if use_ssl:
-        uri_scheme = "https://"
-    else:
-        uri_scheme = "http://"
+    hec = http_event_collector(token, host, "json", name, port, use_ssl)
```

Comment:
Please create a constant `FORMAT_JSON = "json"` and use it here.
```suggestion
    hec = http_event_collector(token, host, FORMAT_JSON, name, port, use_ssl)
```